### DISCUSSION TITLE
Survey on policies regarding AI assistants in your repositories

### DISCUSSION THREAD

**Discussion by markxiaotao** [OWASP/wrongsecrets#1666] [General]
Dear developers,

I hope you’re doing well. I’m conducting a brief 10-question survey (some of which are optional) focusing on the policies around AI assistants in open-source projects. The survey will take no longer than 10 minutes to complete.

We’ve identified that some of the repositories you contribute to have policies or recommendations regarding the use of AI assistants, and your input on related policies would be highly valuable.

We welcome your feedback via this link:
https://forms.gle/a54GqnrD4kLbCKeA9

Thank you for your time! If you have any questions, feel free to reach out.

Best regards,
Tao Xiao
Nara Institute of Science and Technology, Japan

**Comment by bendehaan** [comment]
Hi Tao Xiao,

I've submitted my response. Good luck with your research!

Best,

Ben de Haan

**Reply by markxiaotao** [reply]
Hi Ben, thank you very much for your effort!
